apps/src/aiTutor/interactionsApi.js

```diff
@@ -0,0 +1,18 @@
+import {getAuthenticityToken} from '../util/AuthenticityTokenStore';
```

Comment:
is there a reason this is in js rather than ts?

Reply:
Nope! Just that I started this work before my meetings week TypeScript detour 😁